tests/test_unit.py

```diff
@@ -61,51 +71,21 @@ def __exit__(self, *args):
 
 BUFFER_OUTPUT = True
 
-class ParseTestCase(ppt.ParseResultsAsserts, TestCase):
-    def __init__(self):
-        super().__init__(methodName='_runTest')
-
-    def _runTest(self):
-
-        buffered_stdout = StringIO()
-
-        try:
-            with resetting(sys, 'stdout', 'stderr'):
-                try:
-                    if BUFFER_OUTPUT:
-                        sys.stdout = buffered_stdout
-                        sys.stderr = buffered_stdout
-                    print(">>>> Starting test", str(self))
-                    with ppt.reset_pyparsing_context():
-                        self.runTest()
-
-                finally:
-                    print("<<<< End of test", str(self))
-                    print()
-
-        except Exception as exc:
-            if BUFFER_OUTPUT:
-                print()
-                print(buffered_stdout.getvalue())
-            raise
 
+class Test1_PyparsingTestInit(TestCase):
```

Comment:
The "Test" prefix is required so pyCharm can see these as tests.  The `Test1`, `Test2`, ...  pattern ensures these test classes are run in order.